src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx

```diff
@@ -115,11 +115,38 @@ To host CPMs, your system must meet the minimum requirements for the chosen syst
 
         <tr>
           <td>
-            Disk space
+            Disk size
+          </td>
+
+          <td>
+            A minimum of 50 GiB per host
```

Comment:
@keegoid-nr I believe this is an error but can you confirm real quick?

Reply:
```suggestion
            A minimum of 50 GB per host
```

Reply:
TIL that Gibabytes are a thing. Ignore this!

Reply:
One of my goals is to standardize units. We should be using GiB. It is not an error.
https://en.wikipedia.org/wiki/Binary_prefix#Adoption_by_IEC,_NIST_and_ISO

Reply:
Using powers of 1000 becomes less and less accurate as sizes get larger.
https://en.wikipedia.org/wiki/Binary_prefix#Deviation_between_powers_of_1024_and_powers_of_1000